src/main/kotlin/com/exactpro/th2/codec/openapi/OpenApiCodec.kt

```diff
@@ -180,47 +196,64 @@ class OpenApiCodec(private val dictionary: OpenAPI) : IPipelineCodec {
     private fun decodeBody(header: Message, rawMessage: RawMessage): Message {
         val body = rawMessage.body
 
-        val bodyFormat = header.getList(HEADERS_FIELD)?.first { it.messageValue.getString("name") == "Content-Type" }?.messageValue?.getString("value") ?: "null"
+        val bodyFormat = header.getList(HEADERS_FIELD)?.first { it.messageValue.getString("name") == "Content-Type" }?.messageValue?.getString("value")?.extractType() ?: "null"
+
+        val uri: String
+        val method: String
+        var code = ""
+
+        val pairFound: Pair<UriPattern, PathItem>
```

Comment:
Right now i cant see scenario when we need more than two values for this case

Reply:
The point is not about the amount values, but about readability because `pairFound.first` / `pairFound.second` doesn't give you any info about what's inside